package.json

```diff
@@ -41,9 +41,7 @@
     "@types/node": "12.0.10",
     "standard": "^10.0.3",
     "tape": "^4.5.1",
-    "typescript": "3.5.2"
-  },
-  "dependencies": {
-    "safe-buffer": "^5.0.1"
+    "typescript": "3.5.2",
+    "uint8arrays": "^1.0.0"
```

Comment:
If possible I'd like to avoid having to verify this dependency.

Reply:
Modifying the fixture is fine, or since it's just using to/from hex, maybe pulling over a tiny helper function directly into tests.

Reply:
I've removed the extra dep and added to/from hex string functions to the test.